src/components/common/Panel/index.tsx

```
@@ -1,28 +1,50 @@
-    background: ${getHslaColor(type)};
-  `;
+  const { border, bg } = STYLES[type];
 
   return (
-    // Different margins for x and y axis to account for the vertical spacing
-    // that exists in text elements
-    <div className={`rounded-lg my-8 px-5 pt-4 pb-1 ${bgClass}`}>
-      <div className="font-bold text-xl mb-2">{title}</div>
+    <div
+      className={cn('rounded-lg my-8 px-5 py-4 border-l-4', border, bg)}
+    >
```

ESLint

[error] Replace `⏎······className={cn('rounded-lg·my-8·px-5·py-4·border-l-4',·border,·bg)}⏎····` with `·className={cn('rounded-lg·my-8·px-5·py-4·border-l-4',·border,·bg)}`

(prettier/prettier)